examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -660,20 +144,6 @@ static void InitOTARequestor(void)
 
 extern "C" void app_main()
 {
-    ESP_LOGI(TAG, "All Clusters Demo!");
-
-    /* Print chip information */
-    esp_chip_info_t chip_info;
-    esp_chip_info(&chip_info);
```

Comment:
Ideally for any Espressif demo, we print the chip info and features at the beginning. Why change that?

Reply:
I see other examples like lock-app and lighting-app, they only print demo name at the begining, I believe it's better to keep a consisit style and reduce the length of main. Instead of remove this information, I put it in AppTask so it can still print.